### ISSUE TITLE
Error: Not Acceptable 

### ISSUE THREAD

**Issue by smdjeff** [nfriedly/express-rate-limit#80]
It appears you're only supporting a couple of res.format responses during rate-limiting events. I think this could be rectified by adding a default case that sends only the error code.

Error: Not Acceptable 
Jun 21 12:37:36 staging-cloud app/web.1:     at ServerResponse.res.format (/app/node_modules/express/lib/response.js:675:15) 
Jun 21 12:37:36 staging-cloud app/web.1:     at Object.handler (/app/node_modules/express-rate-limit/lib/express-rate-limit.js:28:15) 
Jun 21 12:37:36 staging-cloud app/web.1:     at /app/node_modules/express-rate-limit/lib/express-rate-limit.js:81:30 
Jun 21 12:37:36 staging-cloud app/web.1:     at MemoryStore.incr (/app/node_modules/express-rate-limit/lib/memory-store.js:12:7) 
Jun 21 12:37:36 staging-cloud app/web.1:     at rateLimit (/app/node_modules/express-rate-limit/lib/express-rate-limit.js:63:23) 
Jun 21 12:37:36 staging-cloud app/web.1:     at Layer.handle [as handle_request]

**Comment by nfriedly** [comment]
Yea, the idea was that you'd override it with a custom `options.handler` method if you wanted to do anything fancy. Although I think the next major version is going to drop `res.format` and just use `res.send`, which should also resolve this issue.

In the meanwhile, I'd recommend something like this:

```js
var limiter = new RateLimit({
  /* [other options] */
  handler: function (req, res, /*next*/) {
      res.status(429).send(/* whatever ... */);
  }
});
```

**Comment by nfriedly** [comment]
This should be fixed now, the default handler in 3.0 is much simpler and does not attempt content negotiation.
